### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by e111077**
DO NOT MERGE

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -105,7 +105,7 @@
   </div>
 
   <script>
-    document.addEventListener('WebComponentsReady', function() {
```
Comment (notwaldorf) [inline demo/index.html:115]: uhoh, did this really change? should we fix this in `webcomponents.js`?

File: paper-slider.html [hunk 2]
```diff
@@ -173,7 +173,8 @@
       }
 
       .transiting > .slider-knob {
-        transition: left 0.08s ease;
+        transition: left 1.08s ease;
+        /*transition: left 0.08s ease;*/
```
Comment (notwaldorf) [inline paper-slider.html]: soooooo why the extra 1s?

File: paper-slider.html [hunk 3]
```diff
@@ -638,6 +641,7 @@
 
       _knobTransitionEnd: function(event) {
         if (event.target === this.$.sliderKnob) {
+          console.log('knobend false');
```
Comment (notwaldorf) [inline paper-slider.html]: ✂️

File: paper-slider.html [hunk 4]
```diff
@@ -622,11 +624,12 @@
         // Therefore, we need to manually update the `transiting` state
 
         if (prevRatio === this.ratio) {
+          console.log('bardown false');
```
Comment (notwaldorf) [inline paper-slider.html]: ✂️

File: test/a11y.html [hunk 5]
```diff
@@ -74,7 +74,7 @@
         assert.isTrue(focusSpy.called);
       });
 
-      a11ySuite('trivialSlider');
+      // a11ySuite('trivialSlider');
```
Comment (notwaldorf) [inline test/a11y.html]: still broken? can you double check, pls? (some changes were merged in a11ySuite)

**Review by notwaldorf — CHANGES_REQUESTED** [review]

File: paper-slider.html (hunk 2 above)
Comment (e111077) [reply paper-slider.html]: this is awkward. I'm pretty sure this was for testing...

File: demo/index.html (hunk 1 above)
Comment (e111077) [reply demo/index.html:115]: yupppp

File: paper-slider.html (hunk 4 above)
Comment (e111077) [reply paper-slider.html]: butts

File: paper-slider.html (hunk 3 above)
Comment (e111077) [reply paper-slider.html]: double butts

File: test/a11y.html (hunk 5 above)
Comment (e111077) [reply test/a11y.html]: no longer borked

**Comment by e111077** [comment]
@notwaldorf done did the things. Tests fail because WCT hates me and `.restore` and stuff

**Review by notwaldorf — APPROVED** [review]

**Comment by notwaldorf** [comment]
@e111077 i'll investigate this document -> window change, but this is 👍  otherwise

**Pull request merged**